tests/Settings/TestExtruderStack.py

```diff
@@ -325,30 +325,6 @@ def test_removeContainer(extruder_stack):
     with pytest.raises(InvalidOperationError):
         extruder_stack.removeContainer(unittest.mock.MagicMock())
 
-##  Tests setting definitions by specifying an ID of a definition that exists.
-def test_setDefinitionByIdExists(extruder_stack, container_registry):
```

Comment:
You have removed all these tests, do we have a plan to make new tests ?

Reply:
No plan so far. The new managers are based on the ContainerRegistry, which is hard to test.

Reply:
ContainerRegistry has quite a lot of tests and fixtures that put some profiles in.